webui/react/src/components/CreateGroupModal.tsx

```diff
@@ -115,7 +115,7 @@ const CreateGroupModalComponent: React.FC<Props> = ({ onClose, users, group }: P
           groupRoles.map((r) => r.id),
         );
         if (!nameUpdated && !usersUpdated && !rolesUpdated) {
-          message.info('No changes to make.');
+          message.info('No changes to save.');
```

Comment:
good eyes (plural, not sure why people say "good eye" singular)

Reply:
The "eye" in "good eye" refers to the "attention or notice" definition of "eye", not the "ocular organ" definition. (This is related to the verb form)

Reply:
🤩 didn't have an emoji for just one eye...